partiql-tests-data/eval/primitives/operators/like.ion

```diff
@@ -1569,6 +1569,32 @@ like::[
       ]
     }
   },
+  {
```

Comment:
I think, adding the same note as the PR's conversation can help understanding _why_ of these in the future.

Reply:
Which note are you referring to?

Reply:
Oh do you mean Josh's comment?

Reply:
Oh no, I mean the description that you've put on the PR: https://github.com/partiql/partiql-tests/pull/49#issue-1560609110 itself.

Reply:
Got it. I'll add a comment for that to the relevant section for null/missing propagation.